src/components/molecular/LoginForm.vue

```diff
@@ -34,6 +30,26 @@ export default new Molecular({
   components: {
     Popup,
   },
+  created() {
+    localStorage.removeItem('sessionToken')
```

Comment:
It might be worth extracting this into a separate file that can be called whenever necessary, and to clarify what is happening here (resetting the browser to avoid data leaks when not logged in).

Reply:
done bffe2d1